Makefile

```diff
@@ -81,7 +81,7 @@ GO_LDFLAGS                   += -X "github.com/mattermost/${APP_NAME}/internal/v
 
 
 # Architectures to build for
-GO_BUILD_PLATFORMS           ?= linux-amd64 linux-arm64 darwin-amd64 darwin-arm64 freebsd-amd64
+GO_BUILD_PLATFORMS           ?= linux-amd64 linux-arm64 freebsd-amd64
```

Comment:
Why are we discontinuing binary builds for darwin-amd64 and darwin-arm64? 
Is this related to providing multi-arch support with arm64 for the docker image published? 

These list is being parsed during the build, and artifacts are getting published here https://github.com/mattermost/mattermost-push-proxy/releases

Reply:
I agree but we do not support this . I do not think anyone is using darwin on production and if they are they should not since we do not support it .

Reply:
Used these builds locally on my workstation for troubleshooting purposes :) 

We can discard them. I was uncertain about the scope of the PR, as it initially appeared to focus on providing multi-architecture builds for docker images. 
This isn't directly relevant to revisiting the generated artifacts. Hence, my inquiry. 👍